comms/dht/src/actor.rs

```diff
@@ -567,31 +567,31 @@ impl DhtActor {
         trace!(target: LOG_TARGET, "Select peers broadcast strategy: {broadcast_strategy}" );
         #[allow(clippy::enum_glob_use)]
         use BroadcastStrategy::*;
-        let peers = match broadcast_strategy {
+        let (peers, only_connected_nodes) = match broadcast_strategy {
```

Comment:
The actual logic is to bypass excluded address filtering after nodes have been selected, not only connected nodes.
```suggestion
        let (peers, bypass_excluded) = match broadcast_strategy {
```